app/src/main/java/net/squanchy/tweets/view/TweetsAdapter.java

```diff
@@ -8,42 +8,66 @@
 
 import com.twitter.sdk.android.core.models.Tweet;
 import com.twitter.sdk.android.tweetui.Timeline;
-import com.twitter.sdk.android.tweetui.internal.TimelineDelegate;
 
 import net.squanchy.R;
 import net.squanchy.tweets.TweetsPageView;
 
-public class TweetsAdapter extends RecyclerView.Adapter<TweetViewHolder> {
+public class TweetsAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
 
     private final Context context;
-    private final TimelineDelegate<Tweet> delegate;
+    private final TwitterRepository repository;
 
     public TweetsAdapter(Timeline<Tweet> timeline, Context context) {
         this.context = context;
```

Comment:
I think you can safely have the `LayoutInflater` instead of the context as a field since the context is used only for that.